src/pages/index.js

```diff
@@ -1,57 +1,107 @@
 import FormValidator from "../components/FormValidator.js";
-import { initialCards, settings } from "../utils/constants.js";
+import { settings } from "../utils/constants.js";
 import Card from "../components/Card.js";
 import "./index.css";
 import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import UserInfo from "../components/UserInfo.js";
 import PopupWithForm from "../components/PopupWithForm.js";
-("../components/PopupWithForm.js");
+import Api from "../components/Api.js";
+import PopupConfirmation from "../components/PopupConfirmation.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "967eb4d1-2e88-4a75-bdf4-9be2dee67768",
+    "Content-Type": "application/json",
+  },
+});
 
-/* Elements */
+api.getProfileInfo().then(({ name, about, avatar }) => {
+  userInfo.setUserInfo({ name, about });
+  userInfo.setAvatar({ avatar });
+});
 
+api.getInitialCards().then((cardData) => {
+  const section = new Section({ items: cardData, renderer }, ".cards__list");
+  section.renderItems();
+});
+
+/* Elements */
+const imageEditButton = document.querySelector(".profile__image-edit-icon");
 const profileEditButton = document.querySelector("#profile-edit-button");
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const addCardModal = document.querySelector("#card-add-modal");
+const profileImageModal = document.querySelector("#avatar-modal");
 const addNewCardButton = document.querySelector("#card-add-button");
 const profileTitleInput = document.querySelector("#profile-title-input");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
+const profileImageForm = profileImageModal.querySelector("#avatar-edit-form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms